frontend_tests/node_tests/composebox_typeahead.js

```diff
@@ -1399,6 +1421,18 @@ run_test('content_highlighter', () => {
     fake_this = { completing: 'something-else' };
     assert(!ct.content_highlighter.call(fake_this));
 
+    // Verify that it calls 'render_typeahead_item'
+    // when completing stream but rendering a topic item.
+    const topic = {stream_id: 1, topic: 'topic', is_topic_item: true};
+
+    fake_this = { completing: 'stream' };
+    th_render_typeahead_item_called = false;
+    typeahead_helper.render_typeahead_item = function (item) {
+        assert.deepEqual(item, {primary: '#Sweden>topic'});
+        th_render_typeahead_item_called = true;
+    };
+    ct.content_highlighter.call(fake_this, topic);
+
     // Verify that all stub functions have been called.
     assert(th_render_typeahead_item_called);
```

Comment:
We are reusing a variable from some previous test, essentially masking that result completely. If that test didn't set `th_render_typeahead_itema_called)` to true, we would never know.

Reply:
fixed